config/yurt-controller/crd/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+# This kustomization.yaml is not intended to be run by itself,
```

Comment:
at the same time, we should not maintain kustomization files in OpenYurt. only use helm charts to support OpenYurt.